### DISCUSSION TITLE
another _app file inside dashboard folder in pages folder for dashboard layout

### DISCUSSION THREAD

**Discussion by Adnan0061** [vercel/next.js#33625] [Help]
How can I make another _app folder for my dashboard layout inside dashboard folder under pages folder.

<img width="119" alt="_app" src="https://user-images.githubusercontent.com/48943431/150914318-dc21b84b-8dc3-4311-bf1c-73a65ae0f335.png">

my plan is to make something like this so that I can make a fixed layout that is for every page inside dashboard.

Is this possible??

**Comment by cajui54** [comment]
I'm with this same problem!
